ibis/backends/duckdb/tests/conftest.py

```diff
@@ -29,14 +48,27 @@ def preload(self):
 
     @property
     def ddl_script(self) -> Iterator[str]:
+        from ibis.backends.base.sql.alchemy.geospatial import geospatial_supported
+
         parquet_dir = self.data_dir / "parquet"
+        geojson_dir = self.data_dir / "geojson"
         for table in TEST_TABLES:
             yield (
                 f"""
                 CREATE OR REPLACE TABLE {table} AS
                 SELECT * FROM read_parquet('{parquet_dir / f'{table}.parquet'}')
                 """
             )
+        if geospatial_supported:
+            for table in TEST_TABLES_GEO:
+                yield (
+                    f"""
+                    INSTALL spatial;
+                    LOAD spatial;
+                    CREATE OR REPLACE TABLE {table} AS
+                    SELECT * FROM st_read('{geojson_dir / f'{table}.geojson'}')
```

Comment:
```suggestion
                    SELECT * FROM st_read('{geojson_dir / table}.geojson')
```
I _think_ this will work and avoids the nested f-string

Reply:
didn't even see that there -- nah, just leave it alone.  it works and it isn't user-facing